templates/test/ci/prow-ci-version-gmsa/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: default
+resources:
+  - ../prow-ci-version
+patchesStrategicMerge:
+  - patches/vnet-peering.yaml
+  - patches/machine-identity.yaml
```

Comment:
there is an empty file above called "templates/test/ci/patches/windows-gmsa-identity.yaml", not sure if that's intentional?

Reply:
it isn't, I think that was from a previous commit